Sources/FoundationInternationalization/Calendar/Calendar.swift

```diff
@@ -1445,8 +1446,10 @@ public struct Calendar : Hashable, Equatable, Sendable {
     ///
     /// Note that a given entire Day within a calendar is not necessarily all in a weekend or not; weekends can start in the middle of a day in some calendars and locales.
     /// - parameter date: The date at which to begin the search.
+    /// - parameter start: Upon return, the starting date of the next weekend if found.
+    /// - parameter interval: Upon return, the duration of the next weekend if found.
     /// - parameter direction: Which direction in time to search. The default value is `.forward`.
-    /// - returns: A `DateInterval`, or nil if the weekend could not be found.
+    /// - returns: `true` if the next weekend could be found, and `false` if there are no such things as weekend in the calendar and its locale.
```

Comment:
I'm not sure it's actually correct to claim that the result has anything to do with the `Locale` of the `Calendar`. It might be best to just say this:

```suggestion
    /// - returns: `true` if the next weekend is found.
```

Reply:
> I'm not sure it's actually correct to claim that the result has anything to do with the `Locale` of the `Calendar`. It might be best to just say this:

Thank you for your suggestion, Tony. Updated the commit.

The [objective-c method](https://developer.apple.com/documentation/foundation/nscalendar/1409905-nextweekendstartdate) refers to `Locale` and `Calendar`, hence my initial commit. Happy to go with a simpler documentation.